libs/utils/utils.js

```diff
@@ -1026,6 +1026,8 @@ async function checkForPageMods() {
 }
 
 async function loadPostLCP(config) {
+  const event = new Event(MILO_EVENTS.POSTLCP);
+  document.dispatchEvent(event);
```

Comment:
As you don't re-use the event
```suggestion
    document.dispatchEvent(new Event(MILO_EVENTS.POSTLCP));
```